### PR TITLE
CCD-965: Remove Temporary Sonar Exclusions cpo-case-payment-orders-api

### PR DESCRIPTION
**Pull request by JamesCollettCGI**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CCD-965

### Change description ###
CCD-965: Remove Temporary Sonar Exclusions cpo-case-payment-orders-api

**Does this PR introduce a breaking change?**
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by JamesCollettCGI** [comment]
**PR build:** https://build.platform.hmcts.net/job/HMCTS_a_to_c/job/cpo-case-payment-orders-api/view/change-requests/job/PR-188/
**PR sonar:** https://sonarcloud.io/summary/new_code?id=uk.gov.hmcts.reform%3Acpo-case-payment-orders-api&pullRequest=188

File: build.gradle [hunk 1]
```diff
@@ -195,7 +195,6 @@ sonarqube {
     property "sonar.projectKey", "uk.gov.hmcts.reform:cpo-case-payment-orders-api"
     property "sonar.coverage.jacoco.xmlReportPaths", "${jacocoTestReport.reports.xml.destination.path}"
     property "sonar.coverage.exclusions", "**/cpo/Application.java"
```
Comment (rajaram-nagarajan) [inline build.gradle:197]: @JamesCollettCGI - please check sonar.coverage.exclusions property as well

Comment (JamesCollettCGI) [reply build.gradle]: I have removed sonar.coverage.exclusions

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

**Pull request merged**